buildconfig/CMake/googletest_wconversion.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/googlemock/include/gmock/gmock-matchers.h b/googlemock/include/gmock/gmock-matchers.h
+index 33b37a7..4f52f35 100644
+--- a/googlemock/include/gmock/gmock-matchers.h
++++ b/googlemock/include/gmock/gmock-matchers.h
+@@ -1620,8 +1620,9 @@ class VariadicMatcher {
+   }
+ 
+  private:
++#pragma GCC diagnostic ignored "-Wconversion"
```

Comment:
Should this be wrapped in an `#ifdef __GNUC__`? I think that would solve the Windows compiler issue.

Reply:
Had to be a bit more creative since clang also defines `__GNUC__`